packages/react/src/index.ts

```diff
@@ -1,100 +1,1399 @@
-export * from './ActivityItem';
-export * from './Autofill';
-export * from './Announced';
-export * from './Breadcrumb';
-export * from './Button';
-export * from './ButtonGrid';
-export * from './Calendar';
-export * from './Callout';
-export * from './Check';
-export * from './Checkbox';
-export * from './ChoiceGroup';
+/* eslint-disable deprecation/deprecation */
```

Comment:
I understand that this was the easiest temporary approach to make pipeline pass but it defeats the purpose and invites contributors to expose unwanted APIs. 

We need to explicitly disable those lint errors where applicable.

Reply:
I have fixed the disables to be per-line.